.github/workflows/check-vulns.yml

```
@@ -67,6 +87,48 @@ jobs:
       max-parallel: 1
     steps:
       - uses: actions/checkout@v4
+      - name: Build labels for issue
+        id: build_labels
+        run: |
+          # Start with base labels array
+          LABELS_ARRAY=()
+          LABELS_ARRAY+=("${{ inputs.nsolidStream }}")
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)